doc/classes/OS.xml

```diff
@@ -216,6 +216,14 @@
 				Not to be confused with [method get_user_data_dir], which returns the [i]project-specific[/i] user data path.
 			</description>
 		</method>
+		<method name="get_distribution_name" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns on Linux and BSD platforms the user OS' distribution name (e.g. Ubuntu, Manjaro, OpenBSD, etc.). For other platforms, the [method get_name] is returned.
+				For Android, the base OS build is returned.
+				Web is not supported.
```

Comment:
```suggestion
				Returns the name of the distribution for Linux and BSD platforms (e.g. Ubuntu, Manjaro, OpenBSD, etc.). Returns the base OS build for Android. Returns the same value as [method get_name] for other platforms.
				[b]Note:[/b] This method is not supported on the web platform.
```

Note that for every new line a new paragraph is created in the documentation. Which is why you shouldn't split the descriptions like that.

Reply:
I know, but e.g. get_user_data_dir() also splits OS-specific returns into paragraphs

Reply:
Yes, I can see that. That's likely not good either, but the proper way to address it would be to add support for lists in the docs. Multiple short paragraphs really don't improve the reading experience in the docs. If you feel strongly about it, feel free to spread it back into many lines for consistency, but keep other suggested changes.

Reply:
Yeah, I had the same issue in an older PR (like 2.5 years ago)… The workaround I was given back then was to put lists into a code block (back then that was in regard to the description, I think). But I don't think this is a good option for methods/members…

will do the other suggested changes